src/components/DataTable.tsx

```
@@ -0,0 +1,61 @@
+                            ))}
+                        </tr>
+                    ))}
+                    </thead>
+                    <tbody className="divide-y divide-[#E9E9E9] bg-white">
+                    {table.getRowModel().rows.length
+                        ? table.getRowModel().rows.map((row) => (
+                            <tr
+                                key={row.id}
+                                className={twMerge('cursor-pointer hover:bg-[#F8F9F8]', row.getIsSelected() && 'bg-[#F8F9F8]')}
+                                onClick={() => {
+                                    onRowClick && onRowClick(row.original);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)